modules/smtp/modules.php

```diff
@@ -1591,7 +1591,7 @@ function smtp_server_dropdown($data, $output_mod, $recip, $selected_id=false) {
                     $res .= 'selected="selected" ';
                 }
                 $res .= 'value="'.$output_mod->html_safe($vals['id']).'">';
-                $res .= $output_mod->html_safe(sprintf("%s - %s", $vals['user'], $vals['name']));
+                $res .= $output_mod->html_safe(($vals['user'] == $vals['name']) ? $vals['user'] : sprintf("%s - %s", $vals['user'], $vals['name']));
```

Comment:
@DKing2222 did you test? The statement you should have modified is a few lines above (at line 1584) which outputs all the server SMTP profiles. The alternative is used when a server does not have a profile!

Reply:
I tested with and without a profile because even without a profile configured there was a repetition which as claimed does not bring any value